community/planning/consensus_runner.md

```diff
@@ -0,0 +1,281 @@
+---
+tags: [Mermaid]
+mermaid: true
+---
+
+# Consensus Runner
+<!--
+  Copyright 2018-2022 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+## Summary
+
+The consensus runner component performs two important operations for a given
+service: fetching the outstanding ConsensusActions to be executed, followed by
+fetching the next available event to be processed via the appropriate consensus
+algorithm.
+
+## Motivation
+
+As the consensus algorithms do not require threads, processing actions and
+events should also not require any threads.  The consensus runner provides a
+general solution for this problem.
+
+## Guide-level Explanation
+
+A ConsensusRunner struct will operate on a given fully-qualified service ID
+(FQSID).  As part of a timer execution, it will fetch the outstanding actions
```

Comment:
Change to FQSI (fully-qualified service identifier) throughout.

Reply:
Fixed